client/components/question.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from "react";
+import React from "react";
+
+export function Question({ alterCounter, questionApi }) {
+  const [loading, setLoading] = useState(true);
+  const [question, setQuestion] = useState({});
+  const [chosen, setChosen] = useState("");
+
+  // Vi føler dette er en litt weird måte å få lastet et nytt spørsmål etter brukeren har svart
+  // Du må gjerne gi tilbakemelding på en bedre løsning :)
+  const [answered, setAnswered] = useState(0);
+
+  useEffect(async () => {
+    setLoading(true);
+    setQuestion(await questionApi.getQuestion());
+    setLoading(false);
+  }, [answered]);
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    const result = await questionApi.checkAnswer(question.id, chosen);
+    if (result.isCorrect) {
+      alterCounter(1);
+    } else {
+      alterCounter(-1);
```

Comment:
Dette er vel en litt rar måte å behandle feil svar på? I oppgaven var det tiltenkt at man sa "du har svart riktig på 7 av 10 spørsmål". (Dere trenger ikke å rette det for innleveringen, altså)